scripts/damlassistant.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2019, Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+
+import json
+import logging
+import os
+import requests
+import signal
+import socket
+import subprocess
+import sys
+import time
+
+DEFAULT_TRIGGER_SERVICE_PORT = 8088
+DEFAULT_SANDBOX_PORT = 6865
+
+logger = logging.getLogger(__name__)
+
+
+def add_trigger_to_service(party, package_id, trigger):
+    logger.info(f"Starting {package_id}:{trigger} as {party}")
+    payload = {'triggerName': f'{package_id}:{trigger}', 'party': party}
+    headers = {"Content-type": "application/json",
+               "Accept": "application/json"}
+    url = f'http://localhost:{DEFAULT_TRIGGER_SERVICE_PORT}/v1/triggers'
+    response = requests.post(url, json=payload, headers=headers)
+    response.raise_for_status()
+
+
+def get_package_id(dar):
+    result = subprocess.run(['daml', 'damlc', 'inspect-dar', '--json', dar], stdout=subprocess.PIPE)
+    if result.returncode != 0:
+        raise RuntimeError(result.stderr)
+    inspect = json.loads(result.stdout)
+    return inspect["main_package_id"]
```

Comment:
Minor, feel free to ignore.
```suggestion
    dar_properties = json.loads(result.stdout)
    return dar_properties["main_package_id"]
```